src/App.tsx

```diff
@@ -1,15 +1,29 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const foundQuery = (value: string, query: string) => (
+  value.toLowerCase().includes(query.toLowerCase().trim())
+);
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const foundMovies = moviesFromServer.filter(movie => (
```

Comment:
It's better to filter movies only when the query has changed. Consider using [useMemo](https://uk.legacy.reactjs.org/docs/hooks-reference.html#usememo) to do this.